src/panels/config/devices/ha-config-device-page.ts

```diff
@@ -198,7 +202,9 @@ export class HaConfigDevicePage extends LitElement {
     }
 
     this._diagnosticDownloadLinks = Math.random();
+    this._deleteButtons = [".."];
```

Comment:
Added as a hack to not keep re-rendering if there are no delete buttons. What would be the correct way?

Reply:
An empty list is truthy.